keyboards/melody96/rules.mk

```
@@ -24,3 +24,4 @@ NKRO_ENABLE = yes		# USB Nkey Rollover - if this doesn't work, see here: https:/
 BACKLIGHT_ENABLE = yes  # Enable keyboard backlight functionality
 AUDIO_ENABLE = no
 RGBLIGHT_ENABLE = yes
+LINK_TIME_OPTIMIZATION_ENABLE = yes # reduce firmware size
```

This can just be shortened to

```suggestion
LTO_ENABLE = yes # reduce firmware size
```